lib/iris/__init__.py

```diff
@@ -103,7 +103,7 @@ def callback(cube, field, filename):
 
 
 # Iris revision.
-__version__ = "3.1.dev0"
+__version__ = "3.1.0rc0"
```

Comment:
I'm confused by this diff. You already changed the version string yesterday. The 3.1.x branch already has 3.1.0rc0(https://github.com/SciTools/iris/blob/v3.1.x/lib/iris/__init__.py#L106) so why is this trying to change is again?

Reply:
I see there are some conflicting files. Maybe try rebasing onto the 3.1.x branch ?

Reply:
Sorry, I uses the previous branch, so I think it's the probably squash+merge.  I'll rebase onto 3.1.x

Reply:
.. yes, I think that fixed it